app/assets/v2/js/pages/leaderboard.js

```diff
@@ -1,6 +1,28 @@
 /* eslint-disable no-invalid-this */
+var technologies = [
+  '.NET', 'ASP .NET', 'Angular', 'Backbone', 'Bootstrap', 'C', 'C#', 'C++', 'CSS', 'CSS3',
```

Comment:
i think this list should be filtered by technologies for which there is actual leaderboard items.  otherwise, each of them will be empty as you scroll through

Reply:
it'll better this way